genie-common-internal/src/main/java/com/netflix/genie/common/internal/jobs/JobConstants.java

```diff
@@ -292,6 +292,16 @@
      */
     public static final String GENIE_REQUESTED_CLUSTER_TAGS_ENV_VAR = "GENIE_REQUESTED_CLUSTER_TAGS";
 
+    /**
+     * Environment variable for the Genie username the job request.
+     */
+    public static final String GENIE_USER_ENV_VAR = "GENIE_USER";
+
+    /**
+     * Environment variable for the Genie user group the job request.
+     */
+    public static final String GENIE_USER_GROUP_ENV_VAR = "GENIE_USER_GROUP";
```

Comment:
do you think this is useful to anyone? I think we only really need the user

Reply:
Adding both for consistency, since they're tightly related (agreed internally nobody is using this..)

Reply:
I think this concept will go away in agent execution so perhaps we shouldn't do it? It doesn't harm anything I just don't want anyone to start relying on it and then pull it out

Reply:
We could keep this as a concept/model and API level as a way to express the user belongs to a specific group of users.